cloud-azure/src/main/resources/templates/arm-v2.ftl

```diff
@@ -525,7 +533,7 @@
                     "name": "${loadBalancer.name}-publicIp",
                     "location": "[parameters('region')]",
                     "sku": {
-                        "name": "Basic"
+                        "name": "Standard"
```

Comment:
I think this wasn't changed earlier for a reason. @brycederriso and @hreeve-cloudera could confirm if this is safe to do

Reply:
The main reason it wasn't changed was because we weren't sure of the impact of doing so, and also because the PR to add Azure load balancers was very large and we didn't want to add additional complexity. However, revisiting this and eventually changing it to standard was always meant to be a follow up item.